tests/console/download_asset.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Downloads ASSET_1 file to, specified by test_data, file location.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler "get_test_suite_data";
+use File::Basename;
+
+sub run {
+    select_console 'root-console';
+    my $file_location    = get_test_suite_data()->{file_location};
```

Comment:
My reasoning for simplifying the upload/download modules to one asset, is that I'm under the impression that we try to add what we currently need and if in the future the functionality is needed somewhere, it's easy to do the modification. I could change the modules to work for more than one assets and of course it would be convenient for the next person that might utilize this to find it ready, but it's quite possible that we will never need it. WDYT?

Reply:
that is right, also we are compressing and archiving, so all could be put in one. make sense.